boost.cmake

```diff
@@ -83,13 +85,13 @@ MACRO(SEARCH_FOR_BOOST)
       ENDIF(NOT PYTHONLIBS_FOUND)
       IF(${PYTHON_VERSION_MAJOR} EQUAL 3) 
         LIST(REMOVE_AT BOOST_COMPONENTS_ ${PYTHON_IN_BOOST_COMPONENTS})
-        IF(APPLE)
+        IF(APPLE OR NETBSD_FOUND)
```

Comment:
So, shouldn't that also apply to `ARCHLINUX_FOUND`?

Reply:
I missed it, but on the other hand, ArchLinux currently provides boost v1.68 so the ELSE(Boost_VERSION VERSION_GREATER 106699) can be ignored

Reply:
Ok, no problem then :) (sorry, I didn't check the bigger scope of this part of the script)